src/ShallowTestWrapper.js

```diff
@@ -2,6 +2,8 @@ import $ from 'cheerio'
 
 import TestWrapper from './TestWrapper'
 
+const getDisplayName = type => type.displayName || type.name || type
```

Comment:
why is this function needed?

Reply:
@rylanc is this PR going to be finished soon?

Reply:
@dschinkel we are on it, polluting this PR with comments like this one does not help